google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryStage.java

```diff
@@ -428,21 +559,21 @@ ExplainQueryStage toPb() {
 
   static QueryStage fromPb(com.google.api.services.bigquery.model.ExplainQueryStage stagePb) {
     Builder builder = new QueryStage.Builder();
```

Comment:
> in the CL's that large, it's difficult to figure out if something was changed or not, or go back and find the source.

I know and I am sorry for these big PRs. I wanted all the changes for one service to be contained in the same PR, to avoid having inconsistent states in master.